code/modules/admin/admin.dm

```diff
@@ -399,7 +399,7 @@ var/global/floorIsLava = 0
 				I.rank = "N/A"
 				update_file = 1
 			dat += "<font color=#008800>[I.content]</font> <i>by [I.author] ([I.rank])</i> on <i><font color=blue>[I.timestamp]</i></font> "
-			if(I.author == usr.key || check_rights(R_PERMISSIONS, show_msg = 0))
+			if(ckey(I.author) == usr.ckey || check_rights(R_PERMISSIONS, show_msg = 0)) // Older notes stored the key of the banning admin.
```

Comment:
> Older

new notes would also use `key`, it wasn't changed

Reply:
I thought this wouldn't be a problem since now, only the "canonical" key would be compared. I will fix it.

Reply:
It's not a problem in the sense that this works, but the comment is misleading.